assets/js/blocks/cart-checkout/mini-cart-contents/inner-blocks/empty-mini-cart-contents-block/frontend.tsx

```diff
@@ -16,12 +17,22 @@ const EmptyMiniCartContentsBlock = ( {
 }: EmptyMiniCartContentsBlockProps ): JSX.Element | null => {
 	const { cartItems, cartIsLoading } = useStoreCart();
 
+	const elementRef = useRef< HTMLDivElement >( null );
+
+	useEffect( () => {
+		elementRef.current?.focus();
+	} );
```

Comment:
This would trigger it on every re-render/update of the component and we want to focus it only on initial mount/render right? Then empty dependency array is required.

Reply:
or if it's based on if cart is loaded then we should add loaded state as dependency and focus when loaded and ref is set :)

Reply:
> This would trigger it on every re-render/update of the component and we want to focus it only on initial mount/render right? Then empty dependency array is required.

Great catch! Thanks for bringing this up!